tutorials/getting_started/item_widgets.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: documentation
+title: Pages - Item Widgets
+---
+
+# Custom Default Item Widgets
+
+In addition to customizing the Overview Page and the individual cards as discussed on the previous page, the way that the individual Point Items are shown, or whether or not they are shown at all, can be customized as well.
```

Comment:
Maybe splitting this up in two short sentences would increasy readability.

Reply:
I reworked it to express the same thoughts with half the number of words. Hopefully it's easier to read now in the latest push.